src/adapters/MeetingsJSONAdapter.js

```diff
@@ -56,7 +57,8 @@ export const ROSTER_CONTROL = 'member-roster';
  *     "remoteAudio": {},
  *     "remoteVideo": {},
  *     "remoteShare": {},
- *     "state": "JOINED"
+ *     "state": "JOINED",
+ *     "showSettings": false
```

Comment:
this needs to be documented in the interfaces as well, can you please create a task?

Reply:
nvm, I added the subtask https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-236053